src/api/config.c

```diff
@@ -713,7 +713,7 @@ static int api_config_patch(struct ftl_conn *api)
 			return send_json_error_free(api, 400,
 			                            "bad_request",
 			                            "This config option can only be set in pihole.toml, not via the API",
-			                            key, true);
+			                            key, true, false);
```

Comment:
Why is this the only one set to `false`?

Reply:
Because I thought it's not very useful to print this to the log. Given that we also print when a config item is forced through environment, we can probably just keep this as well.